kyaml/yaml/merge3/multiplemergekeys_test.go

```diff
@@ -0,0 +1,422 @@
+package merge3_test
+
```

Comment:
can we factor out the boilerplate so that these are easier to read
```
const preamble = `
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-deployment
spec:
  template:
    spec:
      containers:
      - image: test-image
        name: test-deployment
        ports:`
```

```
origin: preamble + `
-- containerPort...
`
```